src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -29,7 +29,7 @@ public enum CustomQueryInterfaceMode
     public static partial class Marshal
     {
 #if FEATURE_COMINTEROP
-        internal static Guid IID_IUnknown = new Guid("00000000-0000-0000-C000-000000000046");
+        internal static Guid IID_IUnknown = new Guid(0, 0, 0, 0xC0, 0, 0, 0, 0, 0, 0, 0x46);
```

Comment:
You can make it `readonly` while you are on it.

Reply:
It's used as a `ref` argument. Should I change the `BindMoniker` method declaration to use `in`?

Reply:
ok